azure-pipelines.yml

```diff
@@ -151,8 +151,13 @@ stages:
 
     - script: |
         pushd sonic-gnmi
-        make check_gotest ENABLE_TRANSLIB_WRITE=y
-      displayName: "Test"
+        make all && make check_memleak ENABLE_TRANSLIB_WRITE=y
```

Comment:
>all [](http://example.com/codeflow?start=13&length=3)

Do you need `ENABLE_TRANSLIB_WRITE` for `make all`? Since the flag is duplicated, can we define it in a central place?

Reply:
I have updated the pipeline.